tests/resources/mlflow-test-plugin/setup.py

```diff
@@ -6,12 +6,19 @@
     version="0.0.1",
     description="Test plugin for MLflow",
     packages=find_packages(),
+    # Require MLflow as a dependency of the plugin, so that plugin users can simply install
```

Comment:
Hm, maybe we should copy this under `examples` & reference that within the guide. I'll go ahead & do that